udata_front/theme/gouvfr/templates/header.html

```diff
@@ -92,10 +92,17 @@
                                     </a>
                                 </li>
                                 <li>
-                                    <a
-                                        href="{{ url_for('security.logout') }}"
-                                        class="fr-btn fr-icon-logout-box-r-line"
-                                    >
+                                    {% if current_user.is_authenticated and current_user.password %}
```

Comment:
I think the `password` is not a good discriminate for a ProConnect session since a user can both use datagouv login and ProConnect.

Reply:
ah yes of course, we talked about that, and in the meantime I changed the code to [store the `id_token` in the session](https://github.com/datagouv/udata-front/pull/504/files#diff-6d88bad001db215ea314bfaf930dfc0a219958455a2c8d0241e3dc47be001a4dR27), so this could be a good check, right?

Reply:
I think so indeed, thank you!